### PR TITLE
Grammar updates to docs customization section

### PR DESCRIPTION
**Pull request by carbide20**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Related tickets | none |
| License | MIT |

### CODE REVIEW

File: docs/customization/translation.rst [hunk 1]
```diff
@@ -15,23 +15,23 @@ For example:
...
-    You can create different files for different locales (languages). For example ``messages.pl.yml`` should hold only polish translations,
-    as they will be visible when the current locale is ``PL``. Check :doc:`Locales </book/locales>` docs for more information.
+    You can create different files for different locales (languages). For example ``messages.pl.yml`` should hold only Polish translations,
+    as they will be visible when the current locale is ``PL``. Check the :doc:`Locales </book/locales>` docs for more information.
 
-3. In this file configure the desired key and give it a translation.
+3. In this file, configure the desired key and give it a translation.
 
-If you would like to change the translation of "Email" into "Username" on the login form you have to
+Example: If you would like to change the translation of "Email" into "Username" on the login form you have to
```
Comment (TheMadeleine) [inline docs/customization/translation.rst]: Do we need this change?

File: docs/customization/controller.rst [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 
-To add your custom actions you need to override controllers. You may bee needing to:
+To add your custom actions you need to override controllers. You may need to:
 
-* add a generic action that will render a list of recommended products with a product on its show page,
+* add a generic action that will render a list of recommended products with a product on its show page.
 * render a partial template that cannot be done via yaml resource action.
 
-How to customize a Resource Controller?
+How to customize a Resource Controller:
```
Comment (carbide20) [inline docs/customization/controller.rst]: No problem! How should I update the pull request, do I submit a new one without the changes you've mentioned?

File: docs/customization/translation.rst (hunk 1 above)
Comment (carbide20) [inline docs/customization/translation.rst]: I agree we don't, I went back and re-read the section and the entire section is one big example, this only being part of it. This change is not needed.

**Comment by carbide20** [comment]
Glad to help out! My team is developing on Sylius now, so I'll be making more contributions in the future. I'm not sure how to go about submitting the changes again, should I make a new pull request? Or is there some way to keep it connected to this pull request?

**Comment by michalmarcinkowski** [comment]
Hi Dan!

Thanks for your contribution, that's great you joined Sylius community! You should be able to find everything you need to in our documentation (see [contributing guide](http://docs.sylius.org/en/latest/contributing/code/patches.html)). 

Just quick tip, create a new branch on your fork for each contribution, PR is associated with a branch, so to apply review changes you should make changes, commit and push them to see the update in your PR.

**Comment by carbide20** [comment]
Great to know, thanks @michalmarcinkowski ! Since I had to create a new branch for these changes, I've created a new PR for the changes, including the updates suggested by @TheMadeleine . This is #6654 . Next time, I will have changes on their own branch to begin with, so I can simply push updates. Thanks!

**Pull request closed without merging**